### PR TITLE
Aki 626

### PR DESCRIPTION
**Pull request by dev-dennis-guye**

### CODE REVIEW

File: rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java [hunk 1]
```diff
@@ -971,7 +971,7 @@ public static VersionType fromString(String x){
         }
     }
     public enum BlockEnum{
-        LATEST("latest");
+        LATEST("latest"),        PENDING("pending"),        EARLIEST("earliest");
```
Comment (AionJayT) [inline rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java]: weird formating

File: rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java [hunk 2]
```diff
@@ -981,6 +981,10 @@ public static BlockEnum fromString(String x){
             if(x==null) throw ParseErrorRPCException.INSTANCE;
             if(x.equals("latest")){
                 return LATEST;
+            }else            if(x.equals("pending")){
```
Comment (AionJayT) [inline rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java]: Are the no-capital strings as the input STANDARD or should we ignore the case issue?

Comment (dev-dennis-guye) [reply rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java]: Good catch.

File: rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java (hunk 1 above)
Comment (dev-dennis-guye) [reply rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java]: This is a result of the template file.

Comment (dev-dennis-guye) [reply rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java]: I've fixed it in the following commit

**Pull request merged**